client/src/components/reviews/ReviewEntry.styles.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
```

Comment:
you can also do `export const ReviewerNameDate = {
  float: 'right'
}` and apply it as a style to any type of html tag